scripts/bert/finetune_classifier.py

```diff
@@ -346,13 +362,64 @@ def train(metric):
                     ','.join([i + ':{:.4f}' for i in metric_nm])
                 logging.info(eval_str.format(epoch_id + 1, batch_id + 1, len(train_data), \
                     step_loss / args.log_interval, \
-                    trainer.learning_rate, *metric_val))
+                    trainer.learning_rate, *metric_val,))
                 step_loss = 0
+
         mx.nd.waitall()
         evaluate(metric)
         toc = time.time()
         logging.info('Time cost={:.1f}s'.format(toc - tic))
-        tic = toc
```

Comment:
why remove this?

Reply:
> 
> 
> why remove this?

at the start of epoch, tic will be reassignment by tic=time.time(), so i think here is invalid, and remove this.